api/v2/openapi.yaml

```
@@ -189,6 +189,50 @@ paths:
           $ref: '#/responses/InternalServerError'
         '400':
           $ref: '#/responses/BadRequest'
+  /alerts/groups:
+    get:
+      tags:
+        - alertgroups
```

```suggestion
        - alertgroup
```

Given that the other endpoints have singular tags, I would suggest the same here.